administration/intro.adoc

```diff
@@ -181,6 +181,9 @@ https://docs.openshift.com/container-platform/3.7/admin_guide/manage_scc.html[SC
 needs to be added prior KubeVirt deployment:
 
 - When you use the stable deployment flow:
+____________________________________
+Important: Starting with version v0.15.0 please replace `kubevirt-privileged` with `kubevirt-handler`
```

Comment:
What does replace mean in this case?

Put differently: Can you please update the complete document (or guide) to use this new service account?

Reply:
replace that service account name in the next line
didn't want to update the original instruction yet, because 0.15.0 is only alpha released atm...?

Reply:
Ok, thanks for the clarification.

Now, what we'll do - I'll create a branch for each KubeVirt release to capture the documentation at that point in time.

Master will then always contain what will be in the next release.

The gap is that we need to publish the documentation for master including all other (or just the previous) release.